frontend/src/components/ResponseQuestion/ScaleQuestionResponse/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { FC } from 'react';
+import { IQuestionResponse } from "../../../models/IQuestionResponse";
+import { IScaleQuestion } from "../../../models/forms/Questions/IQuesion";
+import { Form, Radio } from 'semantic-ui-react';
+import styles from './styles.module.sass';
+
+export const ScaleQuestionResponse: FC<IQuestionResponse<IScaleQuestion>> =
+  ({ question= { id: '23',  details: { min: 1, max: 8, minDescription: 'bad', maxDescription: 'good'} },
+     answerHandler = null }) => {
+  const handleClick = (e, value) => {
+    answerHandler?.(question.id, value);
+  };
+
+  const getVariants = (min: number, max: number) => {
+    const group = question.id;
+    console.log(group);
```

Comment:
console.log("console.log is a debug method, it should not make it's way to repo")

Reply:
Okay, I just forgot about it(